packages/language-service/package.json

```diff
@@ -26,6 +26,9 @@
     "vscode-languageserver-textdocument": "^1.0.11",
     "vscode-uri": "^3.0.8"
   },
+  "devDependencies": {
+    "vscode-languageserver-protocol": "^3.17.5"
```

Comment:
this package is really specific to the language-service, so put it in that package's package.json